VIM_CheatSheet.md

```diff
@@ -51,5 +57,25 @@
 	#Exit a bash session to go back to Vim
 		exit
 
-		
-		
+	#Turn word-wrap on/off
+        :set wrap
+        :set nowrap	
+
+    #Turn line numbers on/off
+        :set number
+        :set nonumber
+
+    #Open multiple files in new tabs (from bash prompt)
+        vim -p <file1> <file2> ...
+
+    #Quick switch to next tab
+        gt
+
+    #Quick switch to previous tab
+        gT
+
+    #Save tab session to file
```

Comment:
Same as the comment earlier, the same spacing as the others. Thank you my brother.